boudicca.base/publisher-event-html/src/main/resources/templates/events/events.hbs

```diff
@@ -104,11 +114,19 @@
     }
 </style>
 
-<div id="eventsContainer" class="events-grid">
-    {{> events/eventsRaw }}
+<div id="eventsContainer">
+    <div class="events-page events-grid" tabindex="-1">
+        {{> events/eventsRaw }}
+    </div>
 </div>
 <div style="text-align: center">
     <button id="loadMoreButton">Mehr laden</button>
+
+    <div id="resultNotFound" style="display: none">
```

Comment:
do we still use this div? i think the js handling this is gone?

Reply:
I'll remove it.